### ISSUE TITLE
Complete Epsilon Example

### ISSUE THREAD

**Issue by barnettwilliam** [mdenet/educationplatform#13]
Update the Epsilon example to exercise all of the Epsilon languages and functions.  Ensure that all panel types and tool functions in the epsilon playground can be created, displayed, and the functions run and results displayed.

**Comment by szschaler** [comment]
Is this now completed by the work done on [mde-docs.github.io](https://mde-docs.github.io/)?

**Comment by arturboronat** [comment]
@kolovos have the software artifacts been copied to the repo mentioned above?

**Comment by kolovos** [comment]
@arturboronat are you referring to the ETL transformations that Nafey implemented?

**Comment by arturboronat** [comment]
Yes: https://mde-docs.github.io/tutorials/etl-flowchart-to-html.html

**Comment by szschaler** [comment]
Cleaning up issues. I'm unsure whether this can be closed? We have a few examples in the examples repo, but the URL above doesn't actually seem to use the platform for Epsilon stuff at the moment. I'm still tempted to close this issue as it's not strictly an EP issue, but happy to take guidance from others...
